REQUIRE

```diff
@@ -7,3 +7,4 @@ ParserCombinator 1.7.3
 Docile 0.5.3
 Clustering 0.4
 Combinatorics
+DataStructures
```

Comment:
Nope. This is a nonstarter. We should be trying to remove dependencies, not add new ones.